README.md

```diff
@@ -1,4 +1,9 @@
 # Henge
+
+## This project is not maintained anymore.
+## We joined  effords with [Kompose](https://github.com/skippbox/kompose) project.
+
```

Comment:
And then remove everything else in the README imo, to defer people from using Henge and rather Kompose :)